### PR TITLE
Update Translating.md

### PR DESCRIPTION
**Pull request by xk730**
我们在中国的主流视频 网站（就像是油管）为中国人准备了教程视频，视频由我亲自制作录制（总共有3集，时长17分钟），旨在帮助更多喜欢进行翻译工作的中国人参与其中。
Our mainstream video website in China (just like YouTube) has prepared tutorial videos for Chinese people. The videos were made and recorded by me (3 episodes in total, 17 minutes in length), aiming to help more Chinese people who like to do translation work participate in them.

### CODE REVIEW

**Comment by xlenstra** [comment]
Are you okay with it if I rewrite the English part, but leave the Chinese part as-is?

File: docs/Other/Translating.md [hunk 1]
```diff
@@ -123,3 +123,10 @@ A great example is from @SpacedOutChicken's mod Deciv. @The Bucketeer made some
 Your warmongering ways are unacceptable to us. = 即使在野蠻的荒土世界，窮兵黷武的行徑還是無法容忍的！
 (English meaning: These warmongering ways are still unaccepable enen in this world of savage)
 ```
+
+Befor the End
```
Comment (JackRainy) [inline docs/Other/Translating.md:127]: Misprint: Befor**e**

File: docs/Other/Translating.md [hunk 2]
```diff
@@ -123,3 +123,10 @@ A great example is from @SpacedOutChicken's mod Deciv. @The Bucketeer made some
...
+
+If you are Chinese, congratulations on your good luck!
+Here is a tutorial video (like a nanny) specially prepared for Chinese translators by unciv Chinese developers.
```
Comment (JackRainy) [inline docs/Other/Translating.md:130]: Shouldn't Unciv start with a capital U ?

**Review by JackRainy — CHANGES_REQUESTED** [review]

**Comment by xk730** [comment]
> 如果我重写英文部分，但保留中文部分不变，你还好吗？



> Are you okay with it if I rewrite the English part, but leave the Chinese part as-is?

okay ，because I am very happy that someone can modify my poor English

File: docs/Other/Translating.md [hunk 3]
```diff
@@ -123,3 +123,10 @@ A great example is from @SpacedOutChicken's mod Deciv. @The Bucketeer made some
 Your warmongering ways are unacceptable to us. = 即使在野蠻的荒土世界，窮兵黷武的行徑還是無法容忍的！
 (English meaning: These warmongering ways are still unaccepable enen in this world of savage)
 ```
+
+Befor the End
+
+If you are Chinese, congratulations on your good luck!
+Here is a tutorial video (like a nanny) specially prepared for Chinese translators by Unciv Chinese developers.
```
Comment (yairm210) [inline docs/Other/Translating.md:130]: Please remove the 'like a nanny'... if we have first-time translators the last thing we want is for them to think that we're looking down on them...

Comment (xk730) [reply docs/Other/Translating.md:130]: OK, I will revise it, but in my Chinese content, according to the Chinese culture, this word means: content like  is very detailed.
I'm sorry I didn't think that he might have ambiguities.

**Comment by xk730** [comment]
replaced by #7743

**Pull request closed without merging**